src/app/settings/open-sources/open-sources.page.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core'
+
+import licenseFile from 'src/assets/licenses.json'
+
+interface LicenseFile {
+  [key: string]: License
+}
+
+interface License {
+  licenses: string
+  path: string
+  repository?: string
+  publisher?: string
+  licenseFile?: string
+}
+
+@Component({
+  selector: 'app-open-sources',
+  templateUrl: './open-sources.page.html',
+  styleUrls: ['./open-sources.page.scss'],
+})
+export class OpenSourcesPage implements OnInit {
+  licenses: LicenseFile = licenseFile
+
+  constructor() {}
+
+  ngOnInit() {}
```

Comment:
See above: If not needed, the implementation of `OnInit` could be removed.

Reply:
Resolved in 42e9b80